lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -277,7 +282,7 @@ sub _send_request {
 
     my $f = $self->loop->new_future;
     $self->redis->_execute(
-        xadd => XADD => ('rpc_requests', '*', $request_data->@*),
+        xadd => XADD => ('general', '*', $request_data->@*),
```

Comment:
This change wasn't mentioned on the Trello card; please update the description.

Reply:
updated